hles/transformation/src/main/scala/org/broadinstitute/monster/dap/RawRecord.scala

```diff
@@ -33,10 +37,13 @@ case class RawRecord(id: Long, fields: Map[String, Array[String]]) {
     * Get the singleton value for an attribute in this record, if one exists.
     *
     * Raises an error if the attribute has multiple values.
+    *
+    * Only unique duplicates will throw an error
+    * If we receive multiple fields with the same value, the head of the set is used
```

Comment:
is it necessarily safe to discard the extra data if we get multiple unique values for a field?

Reply:
I think the way we have this written - we will throw an error from unique duplicates.
This code should only weed out instances with the same value.